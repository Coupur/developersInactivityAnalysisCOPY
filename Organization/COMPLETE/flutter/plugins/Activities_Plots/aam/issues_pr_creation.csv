id,date,creator_login
297256593,2018-02-14 21:46:16,aam
