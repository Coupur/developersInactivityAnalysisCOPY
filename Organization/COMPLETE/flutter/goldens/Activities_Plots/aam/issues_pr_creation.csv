id,date,creator_login
339059526,2018-07-06 20:17:48,aam
339660551,2018-07-10 01:25:16,aam
